src/Splat.Autofac/SplatAutofacExtensions.cs

```diff
@@ -17,13 +17,13 @@ public static class SplatAutofacExtensions
         /// </summary>
         /// <param name="container">Autofac container.</param>
         public static void UseAutofacDependencyResolver(this IContainer container) =>
```

Comment:
If we change the name to be SetLocator() of the extension method, does this still work?

Reply:
Happy to keep them as is as well.